### ISSUE TITLE
Metadata fields

### ISSUE THREAD

**Issue by pathumego** [indicjs/conversion-maps#1]
Great works @asdofindia I think it is better to add some metadata keys to the files so we can retain 
- Name of the map
- A description (Which font and where to find the font and other details)
- License
- Copyright 

THis data is in the Payyans maps as comments and Ive reported there as well (https://github.com/libindic/payyans/issues/6)

**Comment by asdofindia** [comment]
Yes, metadata is indeed required. In fact, that's why I've nested the maps inside a "map" object. Only, as you've identified, it wasn't available easily to be extracted from the payyans maps using a script. So, I just omitted it altogether.

Also, I was (and am) not very sure of the json schema I wanted to follow (or even whether json is required), so I didn't spend a lot of time on adding metadata to the source files.

The way forward is, as you suggested in private communication, to create a schema for such conversion.

**Comment by pathumego** [comment]
Okay so we have to figure out a schema. I guess there are two things to decide here.

1. Source format: think we can use any widespread structured data format for the maps. Using JSON or XML as source format we can parse to any other format for consumption. So the question boils down to what should be the source format. I am inclined towards 'yaml' :) it is structured, easy to read and maintain. Cons are that it rely on spacing character, which might be problematic or confusing to some.

Schema: considering the messy nature of  ISCII and other ASCII like standards. Mappings should explicitly address, Encoding, Font, Script and language.

Since this will also serve as a documentation of encoding standards, I would like to have extensive metadata.

```
Map schema version:
Metadata:
    Map:
        Name:
        Version:
        Description:
        Source:
        Authour:
        Copyright:
        License:
        Script:
   Encoding:
        Name:
        Source:
        Author:
   Font:
           Name:
           url:
           License:

Map:
   Language:
   Language:
```
